src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -1337,19 +1423,37 @@ CLParser::ParseCommandLine(int argc, char** argv)
         (*params_->http_headers)[header] = arg.substr(header.size() + 1);
         break;
       }
-      case 'l':
-        params_->latency_threshold_ms = std::atoi(optarg);
+      case 'l': {
+        std::string latency_threshold_ms{optarg};
+        if (std::stoi(latency_threshold_ms) >= 0) {
+          params_->latency_threshold_ms = std::stoi(latency_threshold_ms);
+        } else {
+          Usage("The latency threshold (in msecs) must be non-negative.");
+        }
         break;
+      }
       case 'c':
         params_->using_old_options = true;
         params_->max_concurrency = std::atoi(optarg);
         break;
-      case 'r':
-        params_->max_trials = std::atoi(optarg);
+      case 'r': {
+        std::string max_trials{optarg};
+        if (std::stoi(max_trials) >= 0) {
+          params_->max_trials = std::stoi(max_trials);
+        } else {
+          Usage("The maximum number of trials must be non-negative.");
+        }
```

Comment:
Are brackets normal/required for case statements? I see that case 'c' above doesn't have them.

Reply:
This is to scope the variable used here, which shares the same name as another variable in another case which was previously unscoped.

Reply:
Any reason why we can't combine case 10 and case 'r'? And 9 and 's', etc?  I don't like duplicated code, and prefer combining vs adding scoping so that the same variables can operate in the same cut and pasted code.